pkg/base1/test-format.ts

```diff
@@ -84,11 +84,11 @@ QUnit.test("format_number", function (assert) {
 
 QUnit.test("format_bytes", function (assert) {
     const checks = [
-        [5, 1000, "5"],
-        [5, 1024, "5"],
-        [999, 1000, "999"],
-        [999, 1024, "999"],
-        [1023, 1024, "1023"],
+        [5, 1000, "5 B"],
```

Comment:
I might have liked to have seen a test here that `0` → `0 B`, but it's covered by integration tests...

Reply:
Ah, that's a good idea! I plan to do a follow-up anyway to clean up our deprecated ways of calling format_bytes() throughout -- I'll do that there!

Reply:
Added in PR #20361